packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -12,10 +12,11 @@ export interface CheckboxProps extends Omit<HTMLProps<HTMLInputElement>, 'value'
   label?: string;
   description?: string;
   value?: boolean;
+  internalValue?: string | number;
```

Comment:
Not sure about this name TBH but couldn't find anything else.

Reply:
Maybe smth like `htmlValue` to make it clear that it's an original HTML attribute (although not sure how clear that really is).

Reply:
Also for a case like this I think we'd add a comment about what this prop is for.